docs/layers-update-flow.md

```diff
@@ -0,0 +1,70 @@
+
+    ```js
+    export const FISHES_PRIORITY = 'fishes-priority';
+    ```
+- `import` the newly created slugs into `constants/layers-urls` and asign it the url string (inside the `LAYERS_URLS` object) from the ArcGIS online service.
+    ```js
+    import {
+      FISHES_PRIORITY
+    } from 'constants/layers-slugs';
+
+    export const LAYERS_URLS = {
+      [FISHES_PRIORITY]: 'https://tiles.arcgis.com/tiles/arcgis/rest/services/Global_marine_fish_prioritisation_TL/MapServer'
+    }
+    ```
+- First of all we need to know if the type of layer we want to add is already present on the `LAYER_TYPES` constant on the `constants/layers-config` file.  
+Find [in this link](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-Layer.html) the list of the available layer types on the ArcGIS js API.  
+These are the layer types we are currently using in the platform:
+    ```js
+    const LAYER_TYPES = {
+      FEATURE_LAYER: 'FeatureLayer',
+      TILE_LAYER: 'TileLayer',
+      VECTOR_TILE_LAYER: 'VectorTileLayer'
+    }
+    ```
+- Now we are ready to add the layer to the `layerConfig` on `constants/layers-config`. We first need to `import` the layer slug and layers urls to then create the entry for the layer on the config object. 
+    ```js
+    import {
+      FISHES_PRIORITY
+    } from 'constants/layers-slugs';
+
+    export const layersConfig = {
+      [FISHES_PRIORITY]: {
+        title: FISHES_PRIORITY,
+        slug: FISHES_PRIORITY,
+        type: LAYER_TYPES.TILE_LAYER,
+        url: LAYERS_URLS[FISHES_PRIORITY],
+        bbox: null
+      }
+    }
+    ```
+- Note that there's a `bbox` (bounding box) key on the layer object. This field serves to tell the app that this layer is restricted to a certain area of the globe. By providing a set of coordinates to this `bbox` whenever the layer is added to the map the camera will _travel_ to the specified location. Serve as an example the _hummingbirds rarity_ layer config , restricted to the Americas:
+    ```js
+    [HUMMINGBIRDS_RARITY]: {
+      title: HUMMINGBIRDS_RARITY,
+      slug: HUMMINGBIRDS_RARITY,
+      type: LAYER_TYPES.TILE_LAYER,
+      url: LAYERS_URLS[HUMMINGBIRDS_RARITY],
+      bbox: [-164,-40,-35,56]
+    }
+    ```
+- The last step for a layer to be ready to be usede on the platform is to add the needed configuration to be consumed by the `legend` component. Diferent legend configuration are created depending on the category of the layer (_biodiversity, _human pressures_ or _protected areas_), and those are stored into the associated file in the `constants` folder. In this case, fishes priority layer, the legend config to be updated is located on the `constants/biodiversity-layers-constants` file.
+    ```js
+    import {
+      FISHES_PRIORITY
+    } from 'constants/layers-slugs';
+
+    export const legendConfigs = {
+      [FISHES_PRIORITY]: {
+        ...defaultGradientConfig,
+        title: "Fishes priority"
+      }
+    }
+    ```
```

Comment:
```suggestion
    ```
- Once the layer is ready to be used by the platform some changes in the UI are needed. Change the size of the sidebar card in `src/components/biodiversity-sidebar-card/biodiversity-sidebar-card-styles.module.scss` so the new element is visible. To have the opening effect the size of the side bar is provided manually.  
```